.travis.yml

```diff
@@ -1,31 +0,0 @@
-language: node_js
-
-node_js:
-  # FIXME: node v10 has been deprecated, but we are going to keep its regression for a while
-  - "10"
-  - "12"
-
-branches:
-  only:
-    - master
-
-sudo: false
-
-before_install:
-  - npm install -g grunt-cli
-
-install:
-  - npm install
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - gcc
-      - g++
-  code_climate:
-    repo_token: 3af78c71b999d36106cc3ec8761fa5d664515fef7b8c8698e9d5f091a9d5c5a3
```

Comment:
Not sure how to migrate this... probably we can go without it in GitActions. We can add some similar (like Coveralls?) if we need in the future.

NTC (informative)